WPMedia/fusion-news-theme-blocks
# Remove unused scss, clarify purpose of empty scss file

## Description
Remove unused scss, clarify purpose of empty scss file

## Jira Ticket
- no ticket, just cleanup
- minor performance improvement and code quality

## Acceptance Criteria
- No change in styles to local homepage or news theme css 

## Test Steps
npx fusion start -f -l @wpmedia/default-output-block

- open homepage or other basic page to see if layout looks as expected http://localhost/pf/homepage/?_website=the-sun

## Effect Of Changes
### Before
- removing the default.scss forced the import of the news theme css to not work. see https://corecomponents.arcpublishing.com/pf/all-blocks/?_website=washpost&d=658

### After

![Screen Shot 2021-02-09 at 11 51 08](https://user-images.githubusercontent.com/5950956/107405856-5f9e9380-6acd-11eb-94f6-917dda8b70eb.png)


## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- none 

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added. -> yes! 

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
